skyreader/plot/plotting_tools.py

```diff
@@ -10,18 +10,22 @@
 import numpy as np
 from matplotlib import pyplot as plt
 from matplotlib.axes import Axes  # type: ignore[import]
-from matplotlib.projections import projection_registry  # type: ignore[import]
 from matplotlib.projections.geo import MollweideAxes  # type: ignore[import]
 from matplotlib.ticker import FixedLocator, Formatter  # type: ignore[import]
 from matplotlib.transforms import Affine2D  # type: ignore[import]
 
 matplotlib.use('agg')
 
-def format_fits_header(event_id_tuple, mjd, ra, dec, uncertainty):
+def format_fits_header(event_id_tuple, mjd, ra, dec, uncertainty, llh_map):
     """Prepare some of the relevant event information for a fits file
     header."""
     run_id, event_id, event_type = event_id_tuple
 
+    if llh_map:
+        uncertainty_comment = 'Change in 2LLH based on Wilks theorem'
+    else:
+        uncertainty_comment = 'Probability-per-pixel (all pixels with same area)'
```

Comment:
For LVK maps we typically get per-pixel for the normal healpy maps (but probability density for multi-order coverage, since the pixels are different sizes). I think this is consistent with LVK, and we should just be sure to document it in our public info